src/utils/network_calculations/ybus_calculations.jl

```diff
@@ -141,39 +121,75 @@ function _buildybus(branches, nodes, fixed_admittances)
         _ybus!(ybus, b, num_bus)
 
     end
-
     for fa in fixed_admittances
         _ybus!(ybus, fa, num_bus)
     end
+    return SparseArrays.dropzeros!(ybus)
+end
 
-    return ybus
-
+function _goderya(ybus::SparseArrays.SparseMatrixCSC{ComplexF64, Int64})
```

Comment:
Any reference here? I'm guessing a Docstring here providing a reference will be useful. I'm not familiar with his method.

Reply:
No need, not meant to be exported.